### PR TITLE
Player: Remove pointless notification

### LINKED ISSUE #695: Check for players that can get credits by killing someone on the opposite team
i discovered a bug in a video from dhalucard, he was a detective role and got credits from killing someone or his team got credits by killing someone, he got 0 credits multiple times the way to fix this is making a check if the credits are above 0 or something :D

a screenshot of the video were it happens:
image
image

the timestamp of the video were it happens:
https://youtu.be/gBAL6y8LMas?t=234

### PR DESCRIPTION
**Pull request by Reispfannenfresser**
fixes #695

### CODE REVIEW

File: gamemodes/terrortown/gamemode/server/sv_player.lua [hunk 1]
```diff
@@ -656,10 +659,10 @@ local function CheckCreditAward(victim, attacker)
 				for k = 1, #plys do
 					local ply = plys[k]
 
-					if ply:IsActive() and ply:IsShopper() and ply:IsInTeam(attacker) and not ply:GetSubRoleData().preventKillCredits then
-						ply:AddCredits(amt)
+			if ply:IsActive() and ply:IsShopper() and ply:IsInTeam(attacker) and not ply:GetSubRoleData().preventKillCredits then
+				ply:AddCredits(amt)
 
-						--LANG.Msg(GetRoleTeamFilter(TEAM_TRAITOR, true), "credit_kill_all", {num = amt})
+				--LANG.Msg(GetRoleTeamFilter(TEAM_TRAITOR, true), "credit_kill_all", {num = amt})
```
Comment (TimGoll) [inline gamemodes/terrortown/gamemode/server/sv_player.lua:665]: I think this line could be removed

**Review by TimGoll — APPROVED** [review]

File: gamemodes/terrortown/gamemode/server/sv_player.lua (hunk 1 above)
Comment (Reispfannenfresser) [reply gamemodes/terrortown/gamemode/server/sv_player.lua:665]: I don't think i changed this line and i don't understand why it's highlighted.

A lot of stuff in this function can be removed/rewritten and there are some features that might be worth implementing (allowing credit rewards for any combination of roles and teams)
I decided to only fix the bug (add the check) for now and give this another look at a later point.

Comment (Reispfannenfresser) [reply gamemodes/terrortown/gamemode/server/sv_player.lua:665]: wait, nvm
my diff tool doesn't show whitespace changes

**Review by saibotk — APPROVED** [review]
Thank you :rocket: 

Nice to see another contribution of you :)

**Pull request merged**